components/bl-dashlet-component/src/helpers.js

```diff
@@ -0,0 +1,95 @@
+import { useRef, useState, useEffect, useCallback } from 'react';
+
+export const useDraggable = ({ onDrag, rootRef, initialPosition, dragging }) => {
+  if (dragging) {
```

Comment:
Are you get it from somewhere or write it by yourself? Share link for me if it copied from external source.

Reply:
https://stackoverflow.com/a/39192992